nix/unblob/default.nix

```diff
@@ -101,7 +101,11 @@ let
     ];
 
     pythonRelaxDeps = [
+      "attrs"
+      "cstruct"
+      "dissect.cstruct"
       "plotext"
+      "structlog"
       "yaffshiv"
```

Comment:
I didn't want to do this on this branch, as merging conflicting changes in poetry lock is a pain and it may introduce issues in downstream installations which I'd tackle in a separate PR

Reply:
Also, `cstruct` IMHO is no longer required here, as jefferson will pull it in with its fixed packaging.

Reply:
Understood.